eos/models/model.cc

```diff
@@ -26,16 +26,12 @@
 
 namespace eos
 {
-    Model::~Model()
-    {
-    }
+    Model::~Model() {}
 
-    const std::map<Model::KeyType, Model::ValueType>
-    Model::models
-    {
-        std::make_pair("CKM",       &CKMScanModel::make),
-        std::make_pair("SM",        &StandardModel::make),
-        std::make_pair("WET",       &WilsonScanModel::make),
+    const std::map<Model::KeyType, Model::ValueType> Model::models{
```

Comment:
Do you know why the final `{` is not on a new line?

Reply:
clang-format see this as a plain initializer, which have their braces / parentheses put on the same line as the object name.